### PR TITLE
[ELY-2536] Tool encrypt command creates invalid CLI script for bulk conversion

### PR DESCRIPTION
**Pull request by lvydra**
https://issues.redhat.com/browse/ELY-2536

### CODE REVIEW

**Review by jessicarod7 — APPROVED** [review]
This change looks good, thanks @lvydra!

**Comment by fjuma** [comment]
@Skyllarr Please review on Monday, thanks!

File: tool/src/test/java/org/wildfly/security/tool/FileSystemEncryptRealmCommandTest.java [hunk 1]
```diff
@@ -114,6 +114,12 @@ public void testBulk() throws Exception {
         }
     }
 
+    @Test
+    public void testBulkWithoutNames() throws Exception {
+        String descriptorFileLocation = "./target/test-classes/bulk-encryption-conversion-desc-without-names";
+        runCommand(descriptorFileLocation, 0);
```
Comment (skyllarr) [inline tool/src/test/java/org/wildfly/security/tool/FileSystemEncryptRealmCommandTest.java:120]: @lvydra Looks good! Just a question, the resulting CLI file should be deleted after the test. Can we add a afterclass method to do it?

**Review by skyllarr — APPROVED** [review]
I am approving to not hold this fix. The deletion of generated files can be handled as a subsequent issue

**Comment by fjuma** [comment]
@lvydra Please create a follow up issue and PR to address @Skyllarr's comment. Thanks!

**Review by fjuma — APPROVED** [review]
Thanks @lvydra!

**Pull request merged**